http/src/main/resources/init-resources/azure_vm_init_script.sh

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -e
+#set -e
```

Comment:
Could you uncomment this?

Reply:
yeah, I have that locally, just waiting on this run of test